packages/playwright-test/src/runner.ts

```diff
@@ -221,7 +221,7 @@ export class Runner {
       const allFiles = await collectFiles(project.config.testDir);
       const testMatch = createFileMatcher(project.config.testMatch);
       const testIgnore = createFileMatcher(project.config.testIgnore);
-      const extensions = ['.js', '.ts', '.mjs', ...(process.env.PW_COMPONENT_TESTING ? ['.tsx', '.jsx'] : [])];
+      const extensions = ['.js', '.ts', '.mjs', '.tsx', '.jsx'];
```

Comment:
This could be a breaking behavior, for example if someone has jest tests in the same repo.

Reply:
Not sure how this is breaking, we are under our test folder with our test filter.